### PR TITLE
[docs sprint] update open source quickstarts

### PR DESCRIPTION
**Pull request by adnaans**
Update docs for:

[Playground](https://docs.vocode.dev/open-source/playground)
[Turn-based conversation](https://docs.vocode.dev/open-source/turn-based-conversation)
[Multilingual support](https://docs.vocode.dev/open-source/language-support)
[Connecting Your Agent](https://docs.vocode.dev/open-source/agent-factory)
Delete tracing docs and benchmark code. Also added turn-based and streaming conversation quickstarts to makefile

### CODE REVIEW

File: docs/open-source/agent-factory.mdx [hunk 1]
```diff
@@ -5,29 +5,31 @@ description: 'How to link a custom agent to your app'
...
             return MyActionAgent(
-                agent_config=typing.cast(ActionAgentConfig, self.agent_config),
+                agent_config=agent_config,
                 action_factory=self.action_factory
             )
-        raise Exception("Invalid agent config")
+        elif agent_config.type == "other_agent_type":
+            ...
+        else:
+            raise Exception("Invalid agent config")
```
Comment (ajar98) [inline docs/open-source/agent-factory.mdx:33]: bottom AgentFactory instantiation needs to be updated

**Review by ajar98 — CHANGES_REQUESTED** [review]
1 thing then g2g

File: docs/open-source/agent-factory.mdx (hunk 1 above)
Comment (adnaans) [reply docs/open-source/agent-factory.mdx:33]: No this was intentional (as in they can add others). Would it be better to have just the one agent type?

Comment (ajar98) [reply docs/open-source/agent-factory.mdx:33]: like it no longer can take in an `agent_config`, instantiation needs to be updated

**Review by ajar98 — APPROVED** [review]

**Pull request merged**